### ISSUE TITLE
Matching Issues - Deeper / Blacked

### ISSUE THREAD

**Issue by MajesticMaerlyn** [PAhelper/PhoenixAdult.bundle#605]
Certain videos either do not return results, or return inaccurate ones.

Some examples:

https://members.deeper.com/ella-nova-and-mickey-mod
https://members.deeper.com/honeydrop
https://members.deeper.com/denial
https://members.blacked.com/carter-cruise-obsession-chapter-4
https://members.blacked.com/carter-cruise-obsession-chapter-3
https://members.blacked.com/carter-cruise-obsession-chapter-2

Have attempted several naming schemes, for example in first case:

Deeper - Ella Nova and Mickey Mod (Name Matching)
Deeper - 101089 - Ella Nova and Mickey Mod (Scene ID and Name Matching)
Deeper - 2018.01.01 - Ella Nova and Mickey Mod (Scene Date and Name Matching
Deeper - 2018.01.01 - 101089 - Ella Nova and Mickey Mod (Scene Date, Scene ID and Name Matching)

I am not sure if this is the correct log needed. Noticed similiar issues, but they appear to have been closed.

[com.plexapp.agents.phoenixadult.log](https://github.com/PAhelper/PhoenixAdult.bundle/files/4126313/com.plexapp.agents.phoenixadult.log)

Cheers

**Comment by claygoldfinch** [comment]
Unfortunately not really an issue with the agent, but instead with site search. For example:

https://www.deeper.com/search?q=ella%20nova
https://www.deeper.com/search?q=honeydrop
https://www.deeper.com/search?q=denial

I'll look into implementing something that could resolve some of these issues.

**Comment by MajesticMaerlyn** [comment]
Yep, I wasn't sure if that was the case. I am a novice re. these things  :)

**Comment by DirtyRacer1337** [comment]
Looks like it's site problem, on that [page](https://www.deeper.com/cowgirl?page=6) you can find first video [link](https://www.deeper.com/ella-nova-and-mickey-mod), but it's 404 error
